scripts/start

```diff
@@ -16,6 +16,17 @@ if [[ $UID != 0 ]]; then
     exit 1
 fi
 
+[[ -f /etc/lsb-release ]] && source /etc/lsb-release
+
+if [ "${DISTRIB_ID}"="Ubuntu" ] && [ "${DISTRIB_RELEASE}"="20.04" ]; then
```

Comment:
This throws an error on Debian since `/etc/lsb-release` doesn't exist.

```
./scripts/start: line 21: DISTRIB_ID: unbound variable
```

I suggest using `cat /etc/os-release` or `lsb_release -sci` or a combination to check for the correct OS